frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/ProductServiceBox.tsx

```diff
@@ -48,9 +48,28 @@ const ProductItem = ({
             <Text textStyle="subhead-1" pb="0.25rem" color="secondary.500">
               {product.name}
             </Text>
-            <Text textStyle="caption-1" color="secondary.500">
-              ${centsToDollars(product.amount_cents)}
-            </Text>
+            <Flex gap="1rem">
+              <Flex flexDir="column">
+                <Text textStyle="caption-1" color="content.medium">
+                  Amount
+                </Text>
+                {product.multi_qty && (
+                  <Text textStyle="caption-1" color="content.medium">
+                    Quantity limit
```

Comment:
Hm using of `Flex` to display table-like data might not be suitable here as there is a risk of the cell entries overflowing causing a discontinuation of the pseudo-table. I understand that this is unlikely to happen in the current design, but the next developer might not be aware of this. 

Unless there's some tricky parts which restricts us in using tables, I think we should use try to use `<Table />`.

e.g., R1 C2 overflowing
```
Amount          |   S$1000000000000000
Quantity Limit  |   48.500
                |   between 5 to 10

```

Reply:
[c6ba128](https://github.com/opengovsg/FormSG/pull/6699/commits/c6ba128490a5e9fcd8b33b5ba9869bef1dbfb2ec)

Chakra's tables do have quite a few unnecessary styling to `<Td>` 😅, have overwritten them!